### PR TITLE
[TIMETABLE-128] Added profiles

### LINKED ISSUE #128: Profiles
Remove hardcoded active profile from microservices' configs.
Refactor with environment variable.

### PR DESCRIPTION
**Pull request by JJerome-NM**

### CODE REVIEW

File: timetable-config/src/main/resources/application.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ spring:
   application:
     name: timetable-config-ms
   profiles:
-    active: local, native
+    active: ${TIMETABLE_ACTIVE_PROFILE:local, native}
```
Comment (ArsenOOO7) [inline timetable-config/src/main/resources/application.yml:8]: native?

**Review by ArsenOOO7 — CHANGES_REQUESTED** [review]
question

File: timetable-config/src/main/resources/application.yml (hunk 1 above)
Comment (JJerome-NM) [reply timetable-config/src/main/resources/application.yml:8]: Without "native" exception)
image

Comment (ArsenOOO7) [reply timetable-config/src/main/resources/application.yml:8]: uhm, really, ok

**Review by ArsenOOO7 — APPROVED** [review]

**Pull request merged**